unstructured/documents/elements.py

```diff
@@ -137,6 +137,9 @@ class ElementMetadata:
     parent_id: Optional[Union[str, uuid.UUID, NoID, UUID]] = None
     category_depth: Optional[int] = None
 
+    # Languages in element. TODO(newelh) - More strongly type languages
+    languages: Optional[List[str]] = None
```

Comment:
To answer your question, yes we do currently support "auto" as an an input value to indicate auto language detection. I've been enforcing the type of `languages` to be non-Optional (`List[str] = ["auto"]`), let me know if you have cases where a None value makes sense

Reply:
Not using optional will break metadata functionality. By more strongly type I perhaps meant an enum for each language so we could enforce validation for functions that take a "language" and have stronger typing throughout.

Reply:
update per discussion: all metadata types are optional so this will follow that pattern, but that doesn't affect the typing fot the actual `languages` param